sketch.js

```diff
@@ -28,7 +32,7 @@ function draw() {
   background(0);
   // Draw our background image, then move it at the same speed as the pipes
   image(bgImg, bgX, 0, bgImg.width, height);
-  bgX -= pipes[0].speed;
+  bgX -= speed*0.5;
```

Comment:
I would prefer this change (Parallax, using a speed variable) to be separated from the touch controls PR, please. Also you misspelt speed in your variable declaration.